### ISSUE TITLE
Can't send mail with TLS+SMTP authentication in Coral 5

### ISSUE THREAD

**Issue by felixdro** [coralproject/talk#2680]
**Actual behavior:**
After entering email settings in Coral 5 with smtp server, TLS On, and SMTP authentication activated no email are sent, instead I get the error message below. 

I have tried to using an smtp host from AWS (that I have verified works on a different php application). host: email-smtp.eu-west-1.amazonaws.com. 

I have tried in production on our live server and on localhost development mode with the same error. If I switch to a local testing smtp server without any credentials or TLS it works fine.

I have not tried using sendgrid or any other smtp servers.

Error message from the log:

`job failed to process (clusterNode=master, jobName=mailer, jobID=5, attemptsMade=1)
    err: {
      "message": "INTERNAL_ERROR: 140100153673600:error:1408F10B:SSL routines:ssl3_get_record:wrong version number:../deps/openssl/openssl/ssl/record/ssl3_record.c:332:\n",
      "name": "InternalError",
      "stack": "InternalError: INTERNAL_ERROR: 140100153673600:error:1408F10B:SSL routines:ssl3_get_record:wrong version number:../deps/openssl/openssl/ssl/record/ssl3_record.c:332:\n    Object.jobProcessor (src/core/server/queue/tasks/mailer/processor.ts:308:13)\n    runMicrotasks (<anonymous>)\n    processTicksAndRejections (internal/process/task_queues.js:93:5)\n",
      "id": "30940c70-fa74-11e9-8308-55d68c952933",
      "context": {
        "pub": {},
        "pvt": {
          "reason": "could not send email"
        }
      },
      "originalError": {
        "message": "140100153673600:error:1408F10B:SSL routines:ssl3_get_record:wrong version number:../deps/openssl/openssl/ssl/record/ssl3_record.c:332:\n",
        "name": "Error",
        "stack": ""
      }
    }`

**Versions:**
- Coral: 5.2
- NodeJS: 10.16.3
- NPM: 6.9
- MongoDB: 4.2
- Redis: 3.2
- Browser: Google chrome
- OS: Server Linux, client: different

**Comment by kgardnr** [comment]
Hey @felixdro I believe this is a bug, we've discovered this elsewhere and are evaluating the issue. Can you double check what port you're setting?

Also, if you'd like to contribute a bug fix for this it would be super appreciated!

**Comment by felixdro** [comment]
Hi!

I'm not sure if it's related but out coral instance is running in a docker cluster (AWS ECS) the instance does not have a SSL cert. The SSL cert sits on the Load balancer instead and the traffic between the LB and coral is on port 80 http. 

(User) <- HTTPS:443 -> [Load balancer]  <--- HTTP:80 -- > [Docker cluster with Coral]

The coral instance is set to port 5000 via the env variable.

For the smtp server (email-smtp.eu-west-1.amazonaws.com) I have tried using port 25 and also 587 with the same result.

**Comment by felixdro** [comment]
I don't know why It's not working, a guess is that coral want's to talk to the SMTP-server via SSL and the instance doesn't have an ssl cert. But i'm not sure if that's the case. If I know how to fix it I would gladly help but I think it's out of my knowledge. If I have any other theories I'll let you know @kgardnr  

I hope it's not a big thing to solve!

**Comment by felixdro** [comment]
@kgardnr I got it working on port 465!

When I tried with a sendgrid smtp server, they wrote this about their ports:
> 25, 587 | (for unencrypted/TLS connections)
> 465 | (for SSL connections)

And it didn't work on port 25 or 587 if I had TLS set to On, but when i tried port 465 it worked. And when I switched to our aws smtp server it also worked on port 465! :-) I'm not sure if this always is the case, but I'm glad it worked for us.

I would suggest updating the description (configure-email-smtpPortDescription) to also include this information, if it's correct? 

Instead of "(ex. 25)" it could be something like "(ex. 25, 587. For TLS/SSL encryption use port 465)"?

![coral-email-port-description](https://user-images.githubusercontent.com/15653196/67851077-ed898180-fb09-11e9-9045-bc50e998b100.png)
